docs/source/guides/publishing.rst

```diff
@@ -9,6 +9,8 @@ Note that the Northstar name (``Northstar.Xyz``) is reserved for mods that come
 
 It is recommended to upload the source code of your mod to a public repository like `Github <https://github.com/>`_ to give your users a place to suggest changes and leave feedback in an organised manner.
 
+If the changes your mod makes can be represented in screenshots, gameplay recordings, or GIFs, consider adding those to your README. This way anyone coming across your mod can tell which aspects of the game it changes even before installing it.
+
```

Comment:
It's not immediately obvious to me *how* you would do this. Does thunderstore allow me to link to image files in the package? Do i need to use a third-party image hosting site? Could be nice to elaborate on that and maybe provide an example

Reply:
Yeah, that's a fair point. I found [Fifty.NorthstarSpinner](https://northstar.thunderstore.io/package/Fifty/NorthstarLoadingAnim/) as an example.

In this case, the readme of the downloaded zip looks like:

```markdown
# Fifty.NorthstarSpinner
Changes the loading animation to a northstar themed one

![gif](https://user-images.githubusercontent.com/64418963/152659306-9860f8d7-d7bc-41ed-b2cf-b621f76f610c.gif)
```

i.e. the image was uploaded to Github at some point and then linked to from the Thunderstore readme using plain old markdown. I've never uploaded a mod to Thunderstore before. I'm assuming it takes the README displayed on the webpage from the uploaded zip?

In that case I could give a few example for services to upload the image to (Imgur, Discord, Github) or maybe we just update laundmo's example mod to also include some media. I don't think Thunderstore does video embeds, so in that case one would need to link the video inside the README.

Reply:
Ah alright, that works for me. You could even probably just softlink to the `github.com/username/repo/filename.extension`.
If you put something like that in, i'd say its good for merge,